### ISSUE TITLE
GHA: packages built on Windows don't have proper version

### ISSUE THREAD

**Issue by janusw** [Mapsui/Mapsui#1835]
**Describe the bug**

In the GitHub Actions CI build, the packages built on Mac ("nupkg.mac") have a proper version (determined via `git describe --tags`), but the packages build on Windows don't ("nupkg.win"). They seem to have the version of the last tag/release.

**To Reproduce**

As an example:
1. Go to https://github.com/Mapsui/Mapsui/actions/runs/4150324257
2. Navigate to Artifacts.
3. Download both.
4. Unzip.
5. Inspect the nupkg file names ...

Mac has: Mapsui.4.0.0-beta.7-89-g640c84e1c.nupkg
Win has: Mapsui.4.0.0-beta.7.nupkg

**Expected behavior**

Both package variants should have the same version (the one from Mac).

**Additional context**

It seems the problem is that the Win build uses the `buildpack` script to create the packages:

```
    - name: Build nuget packages
      run: Scripts\buildpack $(git describe --tags)
```

It receives the version as an argument, but apparently that version is not being used anywhere in the script.

**Comment by pauldendulk** [comment]
This is something that I broke when removing the version updater. Here is a fix https://github.com/Mapsui/Mapsui/pull/1836. I tested it by running `scripts\buildpack.bat 4.0.0-beta.9` from the command line and to check if the packages are in Artifacts folder with the 4.0.0-beta.9 version (they were).

**Comment by pauldendulk** [comment]
Over the years the dotnet.yml has grown more complex. I think it can be simplified now that the tools have improved. The dotnet-release-nugets.yml builds a solution with only UnoUwp using msbuild as an exception. Then again the dotnet.yml also does extra checks. Some of which I may not understand yet. For example, just yesterday I had the chance to look at the SamplesMapsuiNugetReferences.ps1, a nifty solution.

**Comment by janusw** [comment]
Fixed via #1836. Thanks, @pauldendulk!
